pkg/util/test/util.go

```diff
@@ -288,6 +288,42 @@ func ValidateStorageCluster(
 
 	pxEndpoint = fmt.Sprintf("%s:%d", svc.Spec.ClusterIP, servicePort)
 
+	if err := validatePodsByName(cluster.Namespace, "portworx-pvc-controller", timeout, interval); err != nil {
+		return err
+	}
+
+	if cluster.Spec.Stork != nil && cluster.Spec.Stork.Enabled {
+		if err := validatePodsByName(cluster.Namespace, "stork", timeout, interval); err != nil {
+			return err
+		}
+		if err := validatePodsByName(cluster.Namespace, "stork-scheduler", timeout, interval); err != nil {
+			return err
```

Comment:
For stork-scheduler and pvc-controller we should check that the image tag is same as the k8s version.

Reply:
Done